components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3560,6 +3575,13 @@ where
             return;
         }
 
+        // When Lightning or BR is importing data to TiKV, their ingest-request may fail because of
+        // region-epoch not matched. So we hope TiKV do not check region size and split region during
+        // importing.
+        if self.ctx.importer.get_mode() == SwitchMode::Import {
```

Comment:
Move it to the line between L3595 and L3596. For most cases, the check will be returned at the check of size_diff_hint, moving the import mode check can save unnecessary atomic load operations which involves memory fence.